odl/phantom/geometric.py

```diff
@@ -330,24 +348,24 @@ def _ellipse_phantom_2d(space, ellipses):
     --------
     shepp_logan : The typical use-case for this function.
     """
-
     # Blank image
     p = np.zeros(space.shape, dtype=space.dtype)
 
+    minp = space.grid.min_pt
```

Comment:
If statement?

Reply:
No, this is exactly the old function, except for stuff moved around.

Reply:
That is, the `min_pt` and `max_pt` arguments need to be removed.

Reply:
Ah I see, removing the arguments solves it.